Library/Homebrew/cmd/update-bash.sh

```diff
@@ -289,10 +301,16 @@ EOS
   # this procedure will be removed in the future if it seems unnecessary
   rename_taps_dir_if_necessary
 
+  DIRS=("$HOMEBREW_LIBRARY"/Taps/*/*)
+  if [[ -z "$SKIP_UPDATE_HOMEBREW_REPOSITOR" ]]
+  then
+    DIRS=("$HOMEBREW_REPOSITORY" "${DIRS[@]}")
+  fi
```

Comment:
Not sure what you mean as a normal list. I went to keep use bash array. In addition, I put git repo checks during array initialization. Therefore, we can avoid unnecessary duplicated IO checks.